azext_edge/edge/_help.py

```diff
@@ -609,6 +611,30 @@ def load_iotops_help():
             az iot ops update --name myinstance -g myresourcegroup --desc "Fabrikam Widget Factory B42"
     """
 
+    helps[
+        "iot ops upgrade"
+    ] = """
+        type: command
+        short-summary: Upgrade an IoT Operations instance to the latest version.
+        long-summary: |
+                      WARNING: This command may fail and require you to delete and re-create your cluster and instance.
```

Comment:
should we add this as warning while running the command?

Reply:
its there already

Reply:
but it's white, should warning in yellow? feel like it's a bit merged into the upgrade contexts

Reply:
ill change the color